modules/ossm-deploy-cluster-wide-control-plane-cli.adoc

```diff
@@ -0,0 +1,73 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-create-smcp.adoc
+////
+:_content-type: PROCEDURE
+[id="ossm-deploy-cluster-wide-control-plane-cli_{context}"]
+= Configuring the control plane for cluster-wide deployment with the CLI
+
+You can configure the `ServiceMeshControlPlane` resource for cluster-wide deployment using the CLI. In this example, `istio-system` is the name of the Service Mesh control plane namespace.
+
+.Prerequisites
+
+* The {SMProductName} Operator must be installed.
```

Comment:
```suggestion
* The {SMProductName} Operator is installed.
```
`must` is implied by the fact that this item is in the prerequisites list

Reply:
Done